### PR TITLE
Feature/alt exporter

### PR DESCRIPTION
**Pull request by keith-ferney**

### CODE REVIEW

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -125,7 +124,7 @@ def course
         token_matches = true
       end
 
-      unless @document && token_matches
+      unless @document && token_matches || 1==2
```
Comment (llamapope) [inline app/controllers/documents_controller.rb]: what is this?

File: app/views/admin/canvas/courses.html.erb [hunk 2]
```diff
@@ -42,7 +42,7 @@
     <% end %>
   </tbody>
 </table>
-<%= paginate @document_meta %>
+<%#TODO figure out why paginate @document_meta is failing %>
```
Comment (llamapope) [inline app/views/admin/canvas/courses.html.erb]: If this isn't the active record collection, then it won't do pagination here

File: db/migrate/20180627220531_add_enable_program_outcome_exporter_to_organization.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AddEnableProgramOutcomeExporterToOrganization < ActiveRecord::Migration[5.1]
+  def change
+    change_table :organizations do |t|
+      t.boolean :enable_program_outcome_exporter
```
Comment (llamapope) [inline db/migrate/20180627220531_add_enable_program_outcome_exporter_to_organization.rb]: I think we want an export type here... I don't want a new column for each new type we create

File: db/schema.rb [hunk 4]
```diff
@@ -227,4 +229,12 @@
     t.index ["versioned_id", "versioned_type"], name: "index_vestal_versions_on_versioned_id_and_versioned_type"
   end
 
+  create_table "workflow_steps", force: :cascade do |t|
```
Comment (llamapope) [inline db/schema.rb]: what is this doing in this set?

File: app/controllers/documents_controller.rb (hunk 1 above)
Comment (keith-ferney) [reply app/controllers/documents_controller.rb]: it was testing i missed

File: app/controllers/admin_controller.rb [hunk 5]
```diff
@@ -110,8 +110,9 @@ def canvas_accounts
   end
 
   def canvas_courses
+    @org = Organization.find_by slug: request.env['SERVER_NAME']
     if params[:show_course_meta]
-        @document_meta = get_document_meta
+        @document_meta = DocumentMeta.where(root_organization_id: @org.id).page(params[:page]).per(3)
```
Comment (llamapope) [inline app/controllers/admin_controller.rb]: params[:per] default to 15 or 25, should be overridable in URL

**Pull request merged**